packages/boxel/addon/components/boxel/tab-bar/index.gts

```diff
@@ -0,0 +1,102 @@
+import Component from '@glimmer/component';
+import { type EmptyObject } from '@ember/component/helper';
+//@ts-expect-error glint does not think this is consumed-but it is consumed in the template https://github.com/typed-ember/glint/issues/374
+import { hash } from '@ember/helper';
+import { LinkTo } from '@ember/routing';
+import { MenuItem } from '@cardstack/boxel/helpers/menu-item';
+import cn from '@cardstack/boxel/helpers/cn';
+import cssVar from '@cardstack/boxel/helpers/css-var';
+import { svgJar } from '@cardstack/boxel/utils/svg-jar';
+import { Link } from 'ember-link';
+import or from 'ember-truth-helpers/helpers/or';
+import '@cardstack/boxel/styles/global.css';
+import './index.css';
+
+interface Signature {
+  Element: HTMLElement;
+  Args: {
+    class?: string;
+    items: MenuItem[];
+    spread?: boolean;
+  };
+  Blocks: EmptyObject
+}
+
+export default class BoxelTabBar extends Component<Signature> {
+  get linkItems() {
+    return this.args.items.filter((item) => item.action instanceof Link).map((item) => ({ link: item.action as Link, menuItem: item, text: item.text }));
+  }
+
+  <template>
+    <div
+      role="tablist"
+      class={{cn "boxel-tab-bar" @class boxel-tab-bar--spread=@spread}}
+      style={{
+        cssVar
+        boxel-tab-bar-background-color="var(--boxel-tab-bar-background-color)"
+        boxel-tab-bar-font="var(--boxel-tab-bar-font)"
+      }}
+      ...attributes
+    >
+      {{#if this.linkItems}}
+        {{#each this.linkItems as |tab|}}
+          {{#if tab.menuItem.inactive}}
+            <div
+              role="tab"
+              data-text={{tab.text}}
```

Comment:
`data-text` should move down to `.boxel-tab-bar__item-text`, too? Could also omit it.

Reply:
re this and the below, I recently (re)learned that `LinkTo` supports `@disabled`, which I think is preferable to having a whole parallel tree, for reasons like this oversight! So I’ve updated with that.